konstantinmuenster/strapi-plugin-rich-text
# chore: #2 adding localization to the editor

### What does it do?
It implements string literals for localization.

### Why is it needed?
Content managers can speak any language other than English.

### How to test it?
Change the preferred adding language on the profile screen, go to the rich text content screen and check the toolbar tooltips.
![image](https://github.com/user-attachments/assets/bf332bf0-9578-4dad-8cf6-a40b695c5ab8)


### Notes

I was unable to make the Revert/Insert Link button work properly:
![image](https://github.com/user-attachments/assets/2894444a-2093-44eb-aa8c-7b642078165f)

I defined the literal properly following the [documentation](https://formatjs.io/docs/core-concepts/icu-syntax#select-format): `"editor.dialog.button.link-confirm": "{action, select, Remove {Eliminar Enlace} Insert {Insertar Enlace}}",`
Here is the function using it, following the [documentation](https://formatjs.io/docs/react-intl/api/#formatmessage):
```ts
          <Button
            onClick={() => onInsertLink()}
            variant={shouldRemove ? "danger-light" : "success-light"}
          >
            {formatMessage(
              {
                id: "editor.dialog.button.link-confirm",
                defaultMessage: "{action} Link",
              },
              {
                action: shouldRemove ? "Remove" : "Insert",
              }
            )}
          </Button>
```

But, as you can see, it does not work.
